src/screens/Passcode/EnterPasscodeContainer.js

```diff
@@ -96,10 +101,15 @@ class EnterPasscodeContainer extends NavigatorComponent<Props & Actions & Settin
     }
   };
 
+  onForgetPasscode = () => {
+    this.props.onForget();
+  }
+
   renderPasscodeScreen() {
     const {
-      navigator, passcodeType, title, onCancel,
+      navigator, passcodeType, title, onCancel, isLoggedIn,
     } = this.props;
+    const shouldShowForget = !isLoggedIn && has(this.props, 'accountId');
```

Comment:
Please use `this.props.accountId != null` or similar instead of `has`.

Reply:
done